packages/frontend/src/components/UserSettingsModal/UserManagementPanel/index.tsx

```diff
@@ -103,35 +103,40 @@ const UserListItem: FC<{
                             </div>
                         </UserInfo>
                     )}
-
-                    <ButtonGroup>
-                        <RoleSelectButton
-                            fill
-                            id="user-role"
-                            options={Object.values(OrganizationMemberRole).map(
-                                (orgMemberRole) => ({
+                    {user.data?.ability?.can(
```

Comment:
I still prefer to use this `boolean condition` rather than the component wrapper, because this is how we use it on the backend